index.js

```diff
@@ -0,0 +1,36 @@
+import * as core from "@actions/core";
+import { NetlifyAPI } from "netlify";
+
+const setProductionBranch = async (netlifyClient, siteId, branch) => {
+  const site = await netlifyClient.updateSite({
+    site_id: siteId,
+    body: {
+      build_settings: {
+        branch: branch,
+      },
+    },
+  });
+
+  if (site.build_settings.repo_branch == branch) {
```

Comment:
super minor thing, i explain in the review on the other repo.

```suggestion
  if (site.build_settings.repo_branch === branch) {
```